### ISSUE TITLE
RX200S CUSTOM LOGO

### ISSUE THREAD

**Issue by kingo26** [arafey/openevic#25]
When using wismec updater to add custom logo this soft bricks my rx200s please help

**Comment by arafey** [comment]
Are you running optimized or generic?

**Comment by kingo26** [comment]
generic, i redone the firmware but everytime i try and apply .bmp logo it reads firmware version 0.00 so have to take batteries out hold right button and replug to computer and flash firmware again

**Comment by tiryaki92** [comment]
Yes i have experienced this too. The device is crashing while loading the logo

**Comment by arafey** [comment]
Also, do you experience this issue on older versions or on myevic as well?

**Comment by kingo26** [comment]
[anonymous-512.zip](https://github.com/arafey/openevic/files/936322/anonymous-512.zip)

i used this logo on previous versions

**Comment by arafey** [comment]
I'll get back to you when I'm at my computer. Responding from my phone now so I can't actually do anything. Should be at some point later in the day or tomorrow.

**Comment by kingo26** [comment]
no problem keep us posted

**Comment by arafey** [comment]
I will try to solve this for 0.16, which should be out over the weekend. I don't own one of these devices and this doesn't happen on my device, which complicates figuring it out. If you look at the commits between 0.12 and 0.15 and spot what might be causing this issue, either let me know or submit a pull request.

I have an idea about the cause, but I'll need you to test it. I'll post it here when the test is ready.
